frontend/e2e_frontend.containerfile

```diff
@@ -0,0 +1,14 @@
+FROM cypress/browsers:22.14.0
+
+WORKDIR /usr/src/cpt-dashboard
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+RUN npm install
+#To bundle your app’s source code inside the Docker image, use the COPY instruction:
+COPY . .
+#Your app binds to port 3000 so you’ll use the EXPOSE instruction to have it mapped by the docker daemon:
```

Comment:
"mapped outside the container" might be better, since we're not using docker.

Having to run functional tests inside a "polluted" frontend context still bothers me, but I guess if there's a way to use it externally against a "clean" frontend, we can figure that out later.

Reply:
i'll remove those comments